azure/openshift_module/elasticsearch-machineconfig.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 99-sysctl-elastic
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,dm0ubWF4X21hcF9jb3VudD0yNjIxNDQ=
+        filesystem: root
+        mode: 0644
+        path: /etc/sysctl.d/99-elasticsearch.conf
```

Comment:
does Watson discovery look for the `/etc/sysctl.d/99-elasticsearch.conf` file specifically? I ask because the config of `vm.max_map_count=262144` is already done with the sysctl config. See: https://github.com/IBM/cp4d-deployment/blob/master/azure/openshift_module/sysctl.conf#L11